src/arviz_plots/plots/pairs_focus_plot.py

```diff
@@ -0,0 +1,222 @@
+"""Pair focus plot code."""
+from copy import copy
+from importlib import import_module
+
+import numpy as np
+from arviz_base import rcParams
+from arviz_base.labels import BaseLabeller
+
+from arviz_plots.plot_collection import PlotCollection
+from arviz_plots.plots.utils import (
+    filter_aes,
+    get_group,
+    process_group_variables_coords,
+    set_wrap_layout,
+)
+from arviz_plots.visuals import divergence_scatter, labelled_x, labelled_y, scatter_x
+
+
+def plot_pairs_focus(
+    dt,
+    var_names=None,
+    filter_vars=None,
+    group="posterior",
+    coords=None,
+    sample_dims=None,
+    focus_var=None,
+    focus_var_coords=None,
+    plot_collection=None,
+    backend=None,
+    labeller=None,
+    aes_by_visuals=None,
+    visuals=None,
+    **pc_kwargs,
+):
+    """Plot a fixed variable against other variables in the dataset.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data
+    var_names: str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars: {None, “like”, “regex”}, optional, default=None
+        If None (default), interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    group : str, optional
```

Comment:
consequently, `default "posterior"`. Also remember that you don't have to think much for all these arguments that are common accross most functions. https://arviz-plots.readthedocs.io/en/latest/contributing/new_plot.html#call-signature has a dropdown "Function signature and docstring template" that you can copy to get started and adapt to your function